### DISCUSSION TITLE
求助运行main.py后出现WARNING:  Invalid HTTP request received.

### DISCUSSION THREAD

**Discussion by x772010568** [binary-husky/gpt_academic#1329] [Q&A]
求助运行main.py后出现WARNING:  Invalid HTTP request received.
网页也进不去
![QQ截图20231126165515](https://github.com/binary-husky/gpt_academic/assets/38371839/e33ae93e-9b9f-477b-94d6-74ba033ef2a6)

**Comment by comi-zhang** [comment]
求助，我也遇到了

**Comment by comi-zhang** [comment]
> 求助运行main.py后出现WARNING: Invalid HTTP request received. 网页也进不去 ![QQ截图20231126165515](https://private-user-images.githubusercontent.com/38371839/285639487-e33ae93e-9b9f-477b-94d6-74ba033ef2a6.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQxMzg4MzIsIm5iZiI6MTcxNDEzODUzMiwicGF0aCI6Ii8zODM3MTgzOS8yODU2Mzk0ODctZTMzYWU5M2UtOWI5Zi00NzdiLTk0ZDYtNzRiYTAzM2VmMmE2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI2VDEzMzUzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyOTYxOTExZmI3YjliOGE2ZjJlNzBhOTQ4MTBhMGM4MmM5YWExY2M5ZjQxYjY5MzQyYmYxMWJlZjBlNWUzNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RLK2KEd32N7fuKsha5fyUUoE5M0aJih27U9X-e76YtY)

兄弟解决了吗
